keras_nlp/utils/tensor_utils.py

```diff
@@ -12,7 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+try:
+    import tensorflow as tf
+except ImportError:
+    raise ImportError(
+        "To use `keras_nlp`, please install Tensorflow: `pip install tensorflow`. "
+        "Tensorflow package is required for data preprocessing with any backend."
```

Comment:
Tensorflow package -> The TensorFlow package

Reply:
Done!